### DISCUSSION TITLE
Push to git from cloud project

### DISCUSSION THREAD

**Discussion by lti-equipamiento** [hasura/graphql-engine#8496] [General]
It would be very practical to be able to push changes from cloud to git without having to export everything to local and then pushing. Just a button in cloud platform you click on and sends changes to git specified branch.
Later it could have more options, but to start a button that just do that would be much appreciated.

**Comment by seanonthenet** [comment]
Hi @lti-equipamiento 
I think the current workflow for this would be to connect to the hasura cloud instance from the Hasura CLI console and then having it track your changes which can then be committed to git. I haven't actually seen a SAAS service being able to actually commit to git before as managing pushes, merges and the rest would open up a huge can of worms.
